### ISSUE TITLE
Ship crash: "lmdb: write failed: MDB_MAP_FULL"

### ISSUE THREAD

**Issue by yapishu** [urbit/urbit#5698]
~matwet has been chugging along since late 2019 without a breach, and I'm hoping today isn't the day that changes. Sometime in the last day it crashed, and I'm unable to resuscitate it. 

I have tried packing+melding, which complete successfully, but it does not take. The error output indicates it has something to do with a maximum LMDB size, which I'm guessing is the event log?

Here is what it prints:

```
~
urbit 1.8
boot: home is /home/reid/matwet
loom: mapped 2048MB
lite: arvo formula 27e494c5
lite: core 7b144622
lite: final state 7b144622
loom: mapped 2048MB
boot: protected loom
live: loaded: MB/485.359.616
boot: installed 348 jets
vere: checking version compatibility
%unlagging
lmdb: write failed: MDB_MAP_FULL: Environment mapsize limit reached
ames: live on 54321
pier: disk write bail

home: bailing out
Aborted (core dumped)
```

**Describe the bug**

Ship will not start

**Expected behaviour**

Ship boots and runs normally

**System (please supply the following information, if relevant):**

 - OS: Ubuntu 20.04
 - 1.8 ARM runtime, unable to check hashes

**Additional context**

```
Filesystem      Size  Used Avail Use% Mounted on
udev             12G     0   12G   0% /dev
tmpfs           2.4G  1.4M  2.4G   1% /run
/dev/sda1        97G   73G   24G  76% /
```

```
4.0K    matwet/.urb/get
463M    matwet/.urb/chk
4.0K    matwet/.urb/put
61G     matwet/.urb/log
463M    matwet/.urb/bhk
61G     matwet/.urb
61G     matwet
```

**Comment by yapishu** [comment]
I found someone else with this issue and they found the fix:
https://github.com/urbit/urbit/issues/5471#issuecomment-986788661

I had to change two values and recompile the binaries:

https://github.com/urbit/urbit/blob/82c41c942c5df985d431ffe13ffb065c6aad5575/pkg/urbit/vere/disk.c#L748

https://github.com/urbit/urbit/blob/06e5c59635bc9e1d1c2c08ae73203fb12ff28e24/pkg/urbit/daemon/main.c#L561

I just overwrote the 60GB value with the 1TB value beneath it. I think maybe botter built these on a 32bit OS and it defaulted to the lower one?

**Comment by botter-nidnul** [comment]
Just to complete the historical record: I modified these variables because I ran into some platform specific bugs on aarch64 at the time. Those bugs have likely been fixed since then. My experience with aarch64 was that there were a lot of problems with compilers and tooling which have mostly been fixed as the aarch64 platform became more mainstream. 

At the time the idea of aarch64 servers being mainstream was a dream. So just[ fixing the problem by setting the limit to 60GB seemed harmless enough](https://github.com/urbit/urbit/pull/3850#issuecomment-732692365).
